Sources/FoundationEssentials/AttributedString/AttributedString+Guts.swift

```diff
@@ -16,6 +16,8 @@
 import _RopeModule
 #endif
 
+package import FoundationInternals
```

Comment:
I think for this and the rest of the `FoundationInternal` imports, we'll either want to move them into the `!FOUNDATION_FRAMEWORK` section (the `else` block above) or surround it in a `#if canImport(FoundationInternals)`

Reply:
Thanks. Updated